metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/search/service_association_search_test.clj

```diff
@@ -0,0 +1,64 @@
+(ns cmr.metadata-db.int-test.concepts.search.service-association-search-test
+  "Contains integration tests for searching service associations."
+  (:require
+   [clojure.test :refer :all]
+   [cmr.common.util :refer [are3]]
+   [cmr.metadata-db.int-test.utility :as util]))
+
+(use-fixtures :each (util/reset-database-fixture {:provider-id "REG_PROV" :small false}
+                                                 {:provider-id "SMAL_PROV1" :small true}))
+
+(deftest find-service-associations
+  (let [coll1 (util/save-collection "REG_PROV" 1 {:extra-fields {:entry-id "entry-1"
+                                                                 :entry-title "et1"
+                                                                 :version-id "v1"
+                                                                 :short-name "s1"}})
+        coll2 (util/save-collection "REG_PROV" 2 {:extra-fields {:entry-id "entry-2"
+                                                                 :entry-title "et2"
+                                                                 :version-id "v1"
+                                                                 :short-name "s2"}})
+        associated-service (util/create-and-save-service "REG_PROV" 1)
+        serv-association1 (util/create-and-save-service-association coll1 associated-service 1 3)
+        serv-association2 (util/create-and-save-service-association coll2 associated-service 2 2)]
+    (testing "find latest revisions"
+      (are3 [service-associations params]
+        (is (= (set service-associations)
+               (set (->> (util/find-latest-concepts :service-association params)
+                         :concepts
+                         (map #(dissoc % :provider-id :revision-date :transaction-id))))))
```

Comment:
Add :created-at to this.

Reply:
I would rather do this later after this PR  is merged as this branch is building fine.